qemu/tests/multi_nics_verify.py

```diff
@@ -103,3 +103,4 @@ def check_nics_num(expect_c, session):
         nic_interface.append(guest_ip)
     session_srl.close()
     logging.info("All the [ %s ] NICs get IPs." % nics_num)
+    vm.destroy()
```

Comment:
Drop `kill_vm = no` in multi_nics_verify.cfg

Reply:
ok,I will delete it.